pkg/application/app.go

```diff
@@ -602,20 +602,11 @@ func (*Avalanche) writeFile(path string, bytes []byte) error {
 	return os.WriteFile(path, bytes, constants.WriteReadReadPerms)
 }
 
-func (app *Avalanche) LoadConfig() (models.Config, error) {
+func (app *Avalanche) ConfigFileExists(path string) bool {
```

Comment:
avoid passing in a path. check config exists from a new app variable that refers to the config file path, to be updated with app.SetConfigPath